src/helpers/SkeletonHelper.js

```diff
@@ -38,6 +40,16 @@ class SkeletonHelper extends LineSegments {
 
 			}
 
+			if ( options.axesHelperSize > 0 ) {
+
+				const helper = new AxesHelper( options.axesHelperSize );
```

Comment:
I agree, but this is for debug, not for production. Speed isn't critical here. It seems like lots of work to reconstruct manually.

Does BatchedMesh help here perhaps?

Reply:
The problem is that user might use `SkeletonHelper` for all skinned meshes in their scenes. If they have 10 complex skinned meshes with 100 bones, you end up with 1000 draw calls instead of 10. Even helpers should not be that inefficient.

> Does BatchedMesh help here perhaps?

Yes.